Modules/IO/NIFTI/test/itkNiftiReadWriteDirectionTest.cxx

```diff
@@ -112,6 +160,30 @@ itkNiftiReadWriteDirectionTest(int argc, char * argv[])
     return EXIT_FAILURE;
   }
 
+  // should not read the image with non-orthogonal sform
+  ITK_TRY_EXPECT_EXCEPTION(ReadImage<TestImageType>(argv[4], false));
+
+  // check if environment flag is used properly
+  // check without permissive option
+  itksys::SystemTools::PutEnv("ITK_NIFTI_SFORM_PERMISSIVE=NO");
+  ITK_TRY_EXPECT_EXCEPTION(itk::ReadImage<TestImageType>(argv[4]));
+  // check with permissive option
+  itksys::SystemTools::PutEnv("ITK_NIFTI_SFORM_PERMISSIVE=YES");
+  ITK_TRY_EXPECT_NO_EXCEPTION(itk::ReadImage<TestImageType>(argv[4]));
+  // check with permissive option, when nothing is specified
+  itksys::SystemTools::PutEnv("ITK_NIFTI_SFORM_PERMISSIVE");
+  ITK_TRY_EXPECT_NO_EXCEPTION(itk::ReadImage<TestImageType>(argv[4]));
```

Comment:
`itksys::SystemTools::PutEnv("ITK_NIFTI_SFORM_PERMISSIVE");` doesn't work

```
~$ export ITK_NIFTI_SFORM_PERMISSIVE
~$ env | grep ITK_NIFTI_SFORM_PERMISSIVE
~$
```
Probably at least empty value is required for environment variable.

BTW, FYI, another test _itkNiftiReadWriteDirectionSmallVoxelTest_ fails because number of arguments is wrong now.

Reply:
`Export` and `PutEnv` might work differently?

Reply:
> `Export` and `PutEnv` might work differently?

Yes, _export_ doesn't override last value if there is no "="

```
r@deb2:~$ export SOME_VAR=1
r@deb2:~$ export SOME_VAR
r@deb2:~$ env | grep SOME_VAR
SOME_VAR=1
r@deb2:~$ 
```

PutEnv seems to unset the variable if there is no "=". Anyway, test works without there 2 lines.
```
bool SystemTools::PutEnv(const std::string& env)
{
  size_t pos = env.find('=');
  if (pos != std::string::npos) {
    std::string name = env.substr(0, pos);
    return setenv(name.c_str(), env.c_str() + pos + 1, 1) == 0;
  } else {
    return kwsysUnPutEnv(env) == 0;
  }
}
```